charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -43,6 +43,10 @@
       mountPath: /sys/kernel/debug
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: false # Need RW for kprobe_events
+    - name: bpffs
```

Comment:
Should this only be added if NPM/CNM is enabled?

Reply:
Yes, fixed